pom.client.xml

```diff
@@ -967,6 +974,6 @@
     <module>sdk/storage/azure-storage-file-share</module>
     <module>sdk/storage/azure-storage-file-datalake</module>
     <module>sdk/storage/azure-storage-queue</module>
-    <module>sdk/storage/azure-storage-queue-cryptography</module>
+<!--    <module>sdk/storage/azure-storage-queue-cryptography</module>-->
```

Comment:
Did you mean to comment this out?

Reply:
Yep. There is no code in this module. Alan said it would likely be removed soon, so I'm avoiding build errors by commenting this out.